django/utils/timezone.py

```diff
@@ -3,16 +3,24 @@
 """
 
 import functools
+import warnings
+
+try:
+    import zoneinfo
+except ImportError:
+    from backports import zoneinfo
+
 from contextlib import ContextDecorator
 from datetime import datetime, timedelta, timezone, tzinfo
 
-import pytz
 from asgiref.local import Local
 
 from django.conf import settings
+from django.utils.deprecation import RemovedInDjango50Warning
 
+# RemovedInDjango50Warning: add 'utc' to __all__.
 __all__ = [
-    'utc', 'get_fixed_timezone',
+    'get_fixed_timezone',
```

Comment:
Do we need to remove `utc` from the list?

Reply:
flake8 says yes: 

> django/utils/timezone.py:22:1: F822 undefined name 'utc' in __all__

Reply:
I think that we can put it back in and `# noqa: F822` until Django 5.0.

It is perfectly valid to have it according to the examples in PEP 562.

I'm just concerned that someone has `from django.utils.timezone import *` and we'll break it. Also, doesn't editor completion stuff often use `__all__`?

Reply:
Okay good proposal @ngnpope I'll go with that.